source/features/deep-reblame.tsx

```diff
@@ -54,14 +54,12 @@ const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number, c
 
 async function redirectToBlameCommit(event: DelegateEvent<MouseEvent, HTMLAnchorElement | HTMLButtonElement>): Promise<void> {
 	const blameElement = event.delegateTarget;
-	if (blameElement instanceof HTMLAnchorElement) {
-		if (event.altKey) {
-			event.preventDefault();
-		} else {
-			return; // Unmodified click on regular link: let it proceed
-		}
+	if (blameElement instanceof HTMLAnchorElement && !event.altKey) {
```

Comment:
I had it like that
https://github.com/sindresorhus/refined-github/pull/2881#pullrequestreview-378184239

Reply:
You had this

https://github.com/sindresorhus/refined-github/blob/bc4938f6a7f79cd63379af31294e5f636de988c2/source/features/deep-blame.tsx#L53-54